### ISSUE TITLE
Incorrect error message when reading data/topol.tpr

### ISSUE THREAD

**Issue by joaomcteixeira** [MDAnalysis/mdacli#43]
After  #28 (at least when I noticed it), the `data/topology` file can't be read:

```
$ mdacli RMSF -s data/topol.tpr -f data/traj.trr -atomgroup all
Warning: No coordinate reader found for data/topol.tpr. Skipping this file.
```

**Comment by PicoCentauri** [comment]
This seems just as warning from MDAnalysis if we parse a file without a coordinates as a GROMACS tpr file. MDAnalysis throws warnings as hell I have to ommit.

**Comment by joaomcteixeira** [comment]
That's true. The command still executes and saves the result. I don't understand the "skip this file" warning then.
We solved the bug but we now need to solve the bad error message.
